packages/@o3r/rules-engine/src/interfaces/action.interfaces.ts

```diff
@@ -12,6 +14,8 @@ export interface ActionUpdateConfigBlock extends ActionBlock {
 
 /**
  * Content of action that updates an asset
+ *
+ * @deprecated use the one exposer by `@o3r/dynamic-content` module. Will be removed in Otter v10.
```

Comment:
same for the one below 
```suggestion
 * @deprecated use the one exposer by `@o3r/dynamic-content` module. Will be removed in Otter v12.
```